.changes/unreleased/tinymce-TINY-11786-2025-03-03.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Improved
+body: Improved scrolling behavior by adding margins when scrolling elements into view, ensuring content isn't at the very edge of the viewport.
```

```suggestion
kind: Improved
body: Scrolling behavior by adding margins when scrolling elements into view, ensuring content isn't at the very edge of the viewport.
```

---

But the word `improved` is also implied at the beginning of the sentence because it's under an "improved" heading.

```suggestion
body: `editor.selection.scrollIntoView()` now pads the target scroll area with a small margin, ensuring content doesn't sit at the very edge of the viewport.
```